lib/internal/Magento/Framework/Mview/etc/mview.xsd

```diff
@@ -46,6 +46,7 @@
         <xs:attribute name="id" type="xs:string" use="required" />
         <xs:attribute name="class" type="classType" use="required" />
         <xs:attribute name="group" type="xs:string" use="required" />
+        <xs:attribute name="iterator" type="classType" default="Magento\Framework\Mview\View\LegacyChangeLogBatchIterator" />
```

Comment:
I don't see `Magento\Framework\Mview\View\LegacyChangeLogBatchIterator` in codebase

Reply:
This is my mistake